package.json

```diff
@@ -11,7 +11,8 @@
     "list-licenses": "license-checker --production --csv --out LICENSE_DEPENDENCIES_ALL",
     "check-licenses": "license-checker --production --failOn \"AGPL-1.0-only; AGPL-1.0-or-later; AGPL-3.0-only; AGPL-3.0-or-later; Beerware; CC-BY-NC-1.0; CC-BY-NC-2.0; CC-BY-NC-2.5; CC-BY-NC-3.0; CC-BY-NC-4.0; CC-BY-NC-ND-1.0; CC-BY-NC-ND-2.0; CC-BY-NC-ND-2.5; CC-BY-NC-ND-3.0; CC-BY-NC-ND-4.0; CC-BY-NC-SA-1.0; CC-BY-NC-SA-2.0; CC-BY-NC-SA-2.5; CC-BY-NC-SA-3.0; CC-BY-NC-SA-4.0; CPAL-1.0; EUPL-1.0; EUPL-1.1; EUPL-1.1;  GPL-1.0-only; GPL-1.0-or-later; GPL-2.0-only;  GPL-2.0-or-later; GPL-3.0; GPL-3.0-only; GPL-3.0-or-later; SISSL;  SISSL-1.2; WTFPL\"",
     "lint": "eslint --config .eslintrc.js --fix",
-    "build-api-docs": "npx typedoc --out docs/source/api"
+    "build-api-docs": "npx typedoc --out docs/api/source/api",
```

Comment:
So, because source contains index.rst, typedoc seems to have issues outputting to a directory that contains documents not created by typedoc.  
Although, now that I think about it, in the Makefile, during the copy, we can move these to a top-level.

Reply:
Given your absolutely correct comment below, that probably should be done.  That is in the 
Makefile:

	cp -R $(SOURCEDIR)/index.rst $(SOURCECOPYDIR)
	cp -R $(SOURCEDIR)/api/modules/ $(SOURCECOPYDIR)/modules/
	cp -R $(SOURCEDIR)/api/README.md $(SOURCECOPYDIR)/

...

ifdef IS_MAC
	find $(SOURCECOPYDIR)/modules/ -type f -name "*.md" -exec sed -i '' -e 's?\.md#?\.html#?g' {} \;
else
	find $(SOURCECOPYDIR)/modules/ -type f -name "*.md" -exec sed -i  -e 's?\.md#?\.html#?g' {} \;
endif

Then in  the index.rst, we just update the toc for the new directory structure:

.. toctree::
   :glob:
   :titlesonly:

   *
   /modules/*